openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -51,185 +51,18 @@ <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
-            <div class="ui-staff ui-toggle-visibility is--collapsed">
-                <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
-                    <i class="icon fa fa-caret-right" aria-hidden="true"></i>
-                    <span>{% trans "Peer Assessments Completed by This Learner" %}</span>
-                </h2>
-                <div class="ui-toggle-visibility__content">
-                    {% for assessment in submitted_assessments %}
-                        {% with peer_num=forloop.counter %}
-                            <h4 class="title--sub">
-                                {% blocktrans %}
-                                    Assessment {{ peer_num }}:
-                                {% endblocktrans %}
-                            </h4>
-                            <table class="staff-info__status__table" summary="{% trans "Assessment" %}">
-                                <thead>
-                                <tr>
-                                    <th abbr="Criterion" scope="col">{% trans "Criterion" %}</th>
-                                    <th abbr="Selected Option" scope="col">{% trans "Selected Option" %}</th>
-                                    <th abbr="Feedback" scope="col">{% trans "Feedback" %}</th>
-                                    <th abbr="Points" scope="col">{% trans "Points" %}</th>
-                                    <th abbr="Points Possible" scope="col">{% trans "Points Possible" %}</th>
-                                </tr>
-                                </thead>
-
-                                <tbody>
-                                {% for criterion in rubric_criteria %}
-                                    {% for part in assessment.parts %}
-                                        {% if part.option.criterion.name == criterion.name %}
-                                            <tr>
-                                                <td class="label">{{ criterion.label }}</td>
-                                                <td class="value">{{ part.option.label }}</td>
-                                                <td class="value">{{ part.feedback }}</td>
-                                                <td class="value">{{ part.option.points }}</td>
-                                                <td class="value">{{ criterion.total_value }}</td>
-                                            </tr>
-                                        {% endif %}
-                                    {% endfor %}
-                                {% endfor %}
-                                </tbody>
-                            </table>
-                            <h4 class="title--sub">{% trans "Overall Feedback" %}</h4>
-                            <div class="student__answer__display__content">
-                                {{ assessment.feedback|linebreaks }}
```

Comment:
I notice example-based and self are missing this feedback section - will my new approach of "display feedback if it exists" work with this?